packages/cli/package.json

```diff
@@ -2,23 +2,14 @@
   "name": "@chainflip-io/cli",
   "version": "0.0.9",
   "description": "",
-  "main": "./dist/lib.js",
```

Comment:
`exports` is preferred over `main` since node 10: https://stackoverflow.com/a/75097813